Rubberduck.sln

```diff
@@ -149,8 +151,23 @@ Global
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x64.Build.0 = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.ActiveCfg = Release|Any CPU
 		{AC4F1D22-D74B-45FF-AB0C-CC2A104FE023}.Release|x86.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|Any CPU.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|Any CPU.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x64.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x64.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x86.ActiveCfg = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Debug|x86.Build.0 = Debug|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|Any CPU.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|Any CPU.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x64.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x64.Build.0 = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x86.ActiveCfg = Release|Any CPU
+		{43B8B201-6FD9-4D77-BBE8-A0741872C38A}.Release|x86.Build.0 = Release|Any CPU
```

Comment:
(not specific to this PR) What are those? Geez WTF where are all these coming from? How do we fix that?

Reply:
Beats me.